helloworld-springboot/src/main/java/com/example/java/gettingstarted/HelloworldApplication.java

```diff
@@ -29,6 +29,16 @@ public String home() {
     return "Hello World!";
   }
 
+  /**
+   * <a href="https://cloud.google.com/appengine/docs/flexible/java/how-instances-are-managed#health_checking">
+   * App Engine health checking</a> requires responding with 200 to {@code /_ah/health}.
+   */
+  @RequestMapping("/_ah/health")
+  public String healthy() {
+    // Message body required though ignored
+    return "Still surviving.";
```

Comment:
[The docs say](https://cloud.google.com/appengine/docs/flexible/java/how-instances-are-managed#health_checking):
> You can write your own custom health-checking code. It should reply to /_ah/health requests with a HTTP status code 200. The response must include a message body, however, the value of the body is ignored (it can be empty).